Src/Sarif.PatternMatcher.Sdk/Fingerprint.cs

```diff
@@ -227,6 +226,29 @@ public string GetValidationFingerprintHashText(bool jsonFormat = false)
             return ComputeHash(validationFingerprint);
         }
 
+        /// <summary>
+        /// Returns a fingerprint comprised of the minimal information that
+        /// uniquely identifies a credential at a specific point-in-time.
+        /// </summary>
+        /// <returns>The minimal fingerprint that uniquely identifies a credential.</returns>
+        public string GetUniqueSecretFingerprint()
```

Comment:
>Unique [](http://example.com/codeflow?start=25&length=6)

I dont think we need to add Unique here. From the fingerprint perspective, it is just the fingerprint with the secret.

Reply:
The same applies for the next method.